src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php

```diff
@@ -197,4 +209,24 @@ protected function triggerBulk(EventResourcePluginInterface $plugin, array $ids)
 
         $this->eventFacade->triggerBulk($plugin->getEventName(), $eventEntityTransfers);
     }
+
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourcePluginInterface $plugin
+     *
+     * @return void
+     */
+    protected function triggerPluginOnce(EventResourcePluginInterface $plugin): void
+    {
+        $this->eventFacade->trigger($plugin->getEventName(), new EventEntityTransfer());
+    }
+
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourceBulkRepositoryPluginInterface|\Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourceRepositoryPluginInterface $plugin
+     *
+     * @return bool
+     */
+    protected function isPluginWithData($plugin): bool
```

Comment:
this function wrong 
```
EventResourceBulkRepositoryPluginInterface::getData(int $offset, int $limit): array
EventResourceRepositoryPluginInterface::getData(array $ids = []): array
```

Reply:
Thanks Dmitry, updated code / logic by comparing it again with commit  7357927e41587d330b520a078761996c2dcd1454 of file EventResourceRepositoryManager.php (git history). In that commit, `triggerPluginOnce` method is used when a plugin is returning no data.